src/less/apps.less

```diff
@@ -95,6 +95,58 @@ img.icon-enricher {
 /* ----------------------------------------- */
 
 .sfrpg {
+
+    // Bubbles
+    .bubble {
```

Comment:
Why does `.bubble` need to be in both `apps.less` and `items.less`?

Reply:
I moved it there so that it could be accessed by other applications in the future. I wanted to use that styling on the trait selector update, for example.

Reply:
It can be removed for now, if you prefer I only add it in once it's actually being applied to something.

Reply:
In which file is the most up to date CSS? I notice the one in `apps` and `items` is different. I plan take the CSS and put it into a LESS mixin to make it reusable in both item sheets and generic apps

Reply:
If there are any differences, the one in `items` should be taken as the correct one to use in that case.